_data/banking.yml

```
@@ -57,18 +71,25 @@ websites:
       - phone
       - email
     doc: https://www.americaschristiancu.com/ministry/services/online-banking-bill-pay.html
+    regions:
+      - us
 
   - name: Andrews Federal Credit Union
     url: https://www.andrewsfcu.org/
     twitter: AndrewsFederal
     facebook: AndrewsFederal
     img: andrewsfederalcu.png
+    regions:
+      - us
 
   - name: Australia and New Zealand Banking Group
     url: https://www.anz.com.au/
     twitter: ANZ_AU
     facebook: ANZAustralia
     img: anz.png
+    regions:
+      - au
+      - nz
```

```suggestion
    regions:
      - au
```
[ANZ New Zealand](https://www.anz.co.nz/personal/) is a separate domain. Not sure if their login systems are linked or not. ANZ also has a [number of other regions that they serve](https://www.anz.com.au/personal/regions/), some with their own domains, some on their main .com domain.